src/ivoc/ocnoiv1.cpp

```diff
@@ -10,31 +10,172 @@ extern "C" {
 extern void hoc_ret();
 extern void hoc_pushx(double);
 extern void nrn_shape_update();
+extern Object** (*nrnpy_gui_helper_)(const char* name, Object* obj);
```

Comment:
Changes to ocnoiv1.cpp cannot be seen because the file is not compiled in a cmake build and in autotools serves only as a stub to allow oc to be compiled as a stand alone program (which is likely unusable). Same with nrnnoiv.c

Reply:
At least with this branch (and the gui-redirect2 branch) they can be seen when using autotools.

For example, add the line

```
    printf("hello from ocnoiv1's hoc_xpanel\n");
```

to the beginning of hoc_xpanel in ocnoiv1, configure without interviews (this is, after all, a thing that is designed to enable replacing interviews)

```
./configure --with-nrnpython=python3.7 --without-iv
```

Build, install, and launch Python as normal:


```
>>> from neuron import h
>>> h.xpanel("cow")
hello from ocnoiv1's hoc_xpanel
0.0
```

Reply:
Ok! I was definitely mistaken about that. Though I have no idea how that can happen. My curiosity may not be sufficient to give me the energy to figure it out.